[ie/orf:on] Allow downloading of video in segments (#10314)
yt-dlp/yt-dlp: README.md

#### soundcloud
* `formats`: Formats to request from the API. Requested values should be in the format of `{protocol}_{extension}` (omitting the bitrate), e.g. `hls_opus,http_aac`. The `*` character functions as a wildcard, e.g. `*_mp3`, and can passed by itself to request all formats. Known protocols include `http`, `hls` and `hls-aes`; known extensions include `aac`, `opus` and `mp3`. Original `download` formats are always extracted. Default is `http_aac,hls_aac,http_opus,hls_opus,http_mp3,hls_mp3`

> #### orfon

Reviewer: ```suggestion
#### orfon (orf.on)
```

> #### orfon (orf.on)

Author: ```suggestion
#### orfon (orf:on)
```

> * `prefer_segments_playlist`: Some videos are available both as a single file and as a segmented playlist. This option will return the segmented playlist when using `orfon:prefer_segments_playlist`

Reviewer: should also note that `--concat-playlist never` is needed if you want the individual segments

> * `prefer_segments_playlist`: Some videos are available both as a single file and as a segmented playlist. This option will return the segmented playlist when using `orfon:prefer_segments_playlist`. If you want to not concatenate the segments you will also need to use `--concat-playlist never`.

Reviewer: ```suggestion
* `prefer_segments_playlist`: Prefer a playlist of program segments instead of a single complete video when available. If individual segments are desired, use `--concat-playlist never --extractor-args "orfon:prefer_segments_playlist"`
```
